sdcm/wait.py

```diff
@@ -124,3 +125,26 @@ def retry_logger(retry_state: tenacity.RetryCallState) -> None:
     )
 
     return cast(R, retry(func))
+
+
+@contextmanager
+def wait_for_log_lines(node, start_line_patterns, end_line_patterns, start_timeout=60, end_timeout=120):
+    """Waits for given lines patterns to appear in node logs despite exception raised"""
+    start_follower = node.follow_system_log(patterns=start_line_patterns)
+    end_follower = node.follow_system_log(patterns=end_line_patterns)
+    start_time = time.time()
+    try:
+        yield
```

Comment:
we need to test how it looks when exception is being raised, i.e. the case we are timing out or failing.
do we still see the errors despite of the other exception being raise ?

Reply:
sure, I'll add unit-test for this